src/pages/viewer-app/viewer-app.tsx

```diff
@@ -94,6 +96,17 @@ const INITIAL_VIEW_STATE = {
   },
 };
 
+const WarningContainer = styled.div`
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
```

Comment:
Move it to `src\components\common.tsx` to reuse. Rename to something more generic, probably

Reply:
Done.